package-lock.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Please do not commit this file. We use yarn.

Reply:
OK... it will be added on the git ignore